src/widgets/DataTable/Row.js

```diff
@@ -4,21 +4,30 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import { View } from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
 
 /**
  * Renders a row of children as outputted by renderCells render prop
  *
  * @param {object} rowData Data to pass to renderCells callback
  * @param {string|number} rowKey Unique key associated to row
  * @param {object} rowState State to pass to renderCells callBack
+ * @param {func} onPress function to call on pressing the row. See below.
+ * @param {object} viewStyle Style object for the wrapping View component
  * @param {func} renderCells renderProp callBack for rendering cells based on rowData and rowState
  *                          `(rowKey, columnKey) => {...}`
- * @param {object} viewStyle Style object for the wrapping View component
+ * Tap gesture events will be captured in this component for any taps on cells within
```

Comment:
I think a comment like this is usually put in the section before @param list

Reply:
a327c609106a22dc650f14ef549d47c96f01fe8f